scene/main/viewport.cpp

```diff
@@ -3185,6 +3185,19 @@ Viewport::MSAA Viewport::get_msaa() const {
 	return msaa;
 }
 
+void Viewport::set_screen_space_aa(ScreenSpaceAA p_screen_space_aa) {
+
+	ERR_FAIL_INDEX(p_screen_space_aa, 2);
```

Comment:
We get this warning with Clang:
```
scene/main/viewport.cpp:3190:2: error: result of comparison of constant 2 with expression of type 'Viewport::ScreenSpaceAA' is always false [-Werror,-Wtautological-constant-out-of-range-compare]
        ERR_FAIL_INDEX(p_screen_space_aa, 2);
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
./core/error_macros.h:126:42: note: expanded from macro 'ERR_FAIL_INDEX'
        if (unlikely((m_index) < 0 || (m_index) >= (m_size))) {                                                     \
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~
```

Up until now we've worked this around by casting the enum value explicitly:
```suggestion
	ERR_FAIL_INDEX((int)p_screen_space_aa, 2);
```